src/site/_collections/algolia.js

```diff
@@ -0,0 +1,185 @@
+/*
```

Comment:
@MichaelSolati have you read this doc? https://www.algolia.com/doc/guides/sending-and-managing-data/prepare-your-data/how-to/indexing-long-documents/

I found it kind of interesting that they recommend splitting on paragraphs so you never hit the limit (though this would produce a _ton_ of records in the index).